### PR TITLE
Maintain visible analytic and resultlist tabs in the url

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

File: projects/arlas-toolkit/src/lib/components/analytics-board/analytics-board.component.ts [hunk 1]
```diff
@@ -374,4 +382,15 @@ export class AnalyticsBoardComponent implements OnInit, AfterViewInit, OnChanges
...
+  private getParamValue(param: string): string {
+    let paramValue = null;
+    const url = window.location.href;
+    const regex = new RegExp('[?&]' + param + '(=([^&#]*)|&|#|$)');
+    const results = regex.exec(url);
+    if (results && results[2]) {
+      paramValue = results[2];
+    }
+    return paramValue;
+  }
```
Comment (sebbousquet) [inline projects/arlas-toolkit/src/lib/components/analytics-board/analytics-board.component.ts:395]: It seems  that this method is never used ?

File: projects/arlas-toolkit/src/lib/components/analytics-board/analytics-board.component.ts [hunk 2]
```diff
@@ -171,7 +172,11 @@ export class AnalyticsBoardComponent implements OnInit, AfterViewInit, OnChanges
       });
     }
 
-
+    const tab = this.getParamValue('at');
```
Comment (MohamedHamouGisaia) [inline projects/arlas-toolkit/src/lib/components/analytics-board/analytics-board.component.ts:175]: @sebbousquet it is used here

File: projects/arlas-toolkit/src/lib/components/analytics-board/analytics-board.component.ts (hunk 1 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/src/lib/components/analytics-board/analytics-board.component.ts:395]: it is used in line 175

File: projects/arlas-toolkit/src/lib/components/analytics-board/analytics-board.component.ts (hunk 2 above)
Comment (sebbousquet) [reply projects/arlas-toolkit/src/lib/components/analytics-board/analytics-board.component.ts:175]: 👌🏼  my bads

**Review by sebbousquet — APPROVED** [review]

**Pull request merged**